_assets/scripts/run_unit_tests.sh

```diff
@@ -128,7 +128,9 @@ grep -h -v "^mode:" ./**/*.coverage.out >> c.out
 rm -rf ./**/*.coverage.out
 
 if [[ $UNIT_TEST_REPORT_CODECLIMATE == 'true' ]]; then
-	cc-test-reporter after-build --prefix=github.com/status-im/status-go
+  # https://docs.codeclimate.com/docs/jenkins#jenkins-ci-builds
+  GIT_COMMIT=$(git log | grep -m1 -oE '[^ ]+$')
```

Comment:
A comment should include the info itself and not just link, since links can be dead in the future.

>When Jenkins creates a build, it can sometimes run the build on a detached HEAD, causing the commit SHA to be different from the SHA of your branch.

I wonder why our builds would get into this detached head state.